app/src/main/java/de/westnordost/streetcomplete/data/AndroidDatabase.kt

```diff
@@ -65,12 +64,11 @@ class AndroidDatabase(private val db: SQLiteDatabase) : Database {
         groupBy: String?,
         having: String?,
         orderBy: String?,
-        limit: String?,
-        distinct: Boolean,
+        limit: Int?,
```

Comment:
why no distinct?

(yeah, I noticed that the variable is not used, but that just looks like a bug)

Reply:
> the variable is not used

Yes, that's the reason. And since it is also never called with `distinct = true`, I figured that the parameter could go.

Reply:
well, hmm, I'd prefer if it would be passed, instead.

Reply:
Done in [`7aa8f04`](https://github.com/streetcomplete/StreetComplete/pull/6119/commits/7aa8f04ed27cb23e9b75f55dd9076ef01645d4f1).